Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+
+RUN apt-get update && apt-get install -y curl apt-transport-https
+RUN curl -sL https://repos.influxdata.com/influxdb.key | apt-key add -
+RUN echo "deb https://repos.influxdata.com/ubuntu xenial stable" | tee /etc/apt/sources.list.d/influxdb.list
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 0C49F3730359A14518585931BC711F9BA15703C6
+RUN echo "deb [ arch=amd64,arm64 ] http://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.4 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-3.4.list
+
+RUN apt-get update && apt-get install -y influxdb mongodb-org
```

Comment:
You could also clean up the apt cache at the end to keep the image size small:
`rm -rf /var/lib/apt/lists/*`